sematic/plugins/metrics_storage/sql/sql_metrics_storage.py

```diff
@@ -211,7 +219,8 @@ def get_aggregated_metrics(
             if metric_type == MetricType.GAUGE.value:
                 metric_value = float(metric_sum) / (metric_count or 1)
 
-            output.series.append((metric_value, tuple(record[n_basic_fields:])))
+            output_labels = tuple(str(label) for label in record[n_basic_fields:])
```

Comment:
This ensures the labels are always strings--never Decimal/float as they would otherwise have sometimes been when epoch labels were used.